src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/Blocks/footer.html.twig

```diff
@@ -38,7 +38,10 @@
         <i class="material-icons">delete</i>
       </a>
       {{ form_widget(productForm.preview) }}
-      {% endif %}
+      <a class="btn btn-outline-secondary ml-1" href="{{ path('admin_product_form', {'id': productId}) }}">
+        {{ 'Edit product V1'|trans({}, 'Admin.Catalog.Feature')|raw }}
+      </a>
+    {% endif %}
```

Comment:
I think we can indent the whole condition block to improve readability

Reply:
I don't know, I like it this way because:
- it keeps the indentation of HTML code ok compared to the parent
- the `if` block is still indented compared to its content so we identify which content is related to the condition

Anyway I'm ready to change this if another feedback needs to be added, however I'd like not to block the PR for this since it's kind of a pain to get the tests to all succeed because of random sanity failures

But I think it would be great to have clear rules about twig indentation and ideally a tool that automatically detect errors in a Google Action